### DISCUSSION TITLE
Welcome to MaaAshEchoes Discussions!

### DISCUSSION THREAD

**Discussion by moulai** [moulai/MaaAshEchoes#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Agosits** [comment]
你做的好，做的好啊
但是新版本的UI和签到啥的好像没适配会卡住

**Reply by moulai** [reply]
已更新：[v0.2.0](https://github.com/moulai/MaaAshEchoes/releases/tag/v0.2.0) ，适配“瘾”版本
